src/components/Cart/CartItems.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
+import "./Cart.css";
+export const CartItems = () => {
+  const { cartState, cartDispatch } = useCart();
+  const { wishlistState, wishlistDispatch } = useWishlist();
+
+  return (
+    <>
+      <div className="cart-product-container">
+        {cartState.map((product) => {
+          return (
+            <div className="cart-card" key={product._id}>
```

Comment:
- Here can we extract this to a separate component itself as ```ProductCard``` it will help in readability.